src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java

```diff
@@ -29,7 +29,7 @@
 public abstract class BaseUiTestCase extends BaseTestCaseWithBackDoorApiAccess {
 
     /** Indicates if the test-run is to use GodMode. */
-    private static boolean isGodModeEnabled;
+    private static boolean isGodModeEnabled = true;
```

Comment:
@angelosdmg you need to fix it here, as such errors are likely to be caused by the changes you introduced. If you believe there are unrelated errors, feel free to post it here.